lblod/frontend-gelinkt-notuleren
# GN-5166: adjustments to IV fraction tables

### Overview
Ensure that fractions without any seats are not included in the fraction tables.

##### connected issues and PRs:
[GN-5166](https://binnenland.atlassian.net/browse/GN-5166)

### Setup
None

### How to test/reproduce
- Start the app
- Log in as a municipality where there are fractions without any seats (check in LMB)
- Open/create an IV
- Ensure the 'IVGR5-LMB-2-grootte-fracties' is included in an AP
- Sync the meeting
- Ensure the 'IVGR5-LMB-2-grootte-fracties' only contains fractions with a seat number > 0
- Similar for 'IVRMW2-LMB-1-zetelverdeling'

### Checks PR readiness
